tools/datasets-info-create.py

```diff
@@ -113,15 +113,15 @@ def query(key, comment, multi_line=False, multi_item=False, extra=None):
 base_name = os.path.basename(args.file_name)
 info["name"] = os.path.splitext(base_name)[0]
 
-target = "none"
+target = None
 if data.domain.has_discrete_class:
     target = "categorical"
 elif data.domain.has_continuous_class:
     target = "numeric"
 info["target"] = target
 
 info["instances"] = len(data)
-info["variables"] = len(data.domain.attributes)
+info["variables"] = len(data.domain.attributes) + len(data.domain.metas)
```

Comment:
This does not include the class variables.
Use `len(data.domain.variables) + len(data.domain.metas)`

Reply:
Is skipped on purpose because of the info["target"] field.

Reply:
The target field contains the information about the type of that variable. That does not mean the variable should not be counted in the field reporting the size of the data table. At least that is how it currently works - see e.g. iris, which has 5 variables and not 4.
I am OK with the current system, but if we want to change it, we can open a discussion (or an ORC) and consider changing it everywhere. We just need to be consistent!